torch/testing/_internal/common_methods_invocations.py

```diff
@@ -8350,6 +8459,20 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
     ),
 ]
 
+# Reduction operator database
+reduction_op_db: List[ReductionOpInfo] = [
```

Comment:
Add it to the main ops db but then create a filtered list like unary_ufuncs does for consistency